Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -1206,45 +1206,13 @@ class ScrollView extends React.Component<Props, State> {
               nativeEvent.contentOffset.y +
               nativeEvent.layoutMeasurement.height,
           });
-        } else if (key === 'LEFT_ARROW') {
-          this._handleScrollByKeyDown(event, {
-            x:
-              nativeEvent.contentOffset.x +
-              -(this.props.horizontalLineScroll !== undefined
-                ? this.props.horizontalLineScroll
-                : kMinScrollOffset),
-            y: nativeEvent.contentOffset.y,
-          });
-        } else if (key === 'RIGHT_ARROW') {
-          this._handleScrollByKeyDown(event, {
-            x:
-              nativeEvent.contentOffset.x +
-              (this.props.horizontalLineScroll !== undefined
-                ? this.props.horizontalLineScroll
-                : kMinScrollOffset),
-            y: nativeEvent.contentOffset.y,
-          });
-        } else if (key === 'DOWN_ARROW') {
-          this._handleScrollByKeyDown(event, {
-            x: nativeEvent.contentOffset.x,
-            y:
-              nativeEvent.contentOffset.y +
-              (this.props.verticalLineScroll !== undefined
-                ? this.props.verticalLineScroll
-                : kMinScrollOffset),
-          });
-        } else if (key === 'UP_ARROW') {
-          this._handleScrollByKeyDown(event, {
-            x: nativeEvent.contentOffset.x,
-            y:
-              nativeEvent.contentOffset.y +
-              -(this.props.verticalLineScroll !== undefined
-                ? this.props.verticalLineScroll
-                : kMinScrollOffset),
-          });
+        } else if (key === 'HOME') {
+          this.scrollTo({x: 0, y: 0});
+        } else if (key === 'END') {
+          this.scrollToEnd({animated: true});
```

Comment:
This code (previously broken) would have arrow keys scroll a scrollView line by line. This is not native macOS behavior and felt kinda weird, so I removed it. 

I replaced it with keyboard handlers for the Home and End Keys. Why isn't native just doing that for us? Because `RCTScrollView` explicitly sends the keyboard event to JS instead of handling it natively. That can be fixed as a followup / separate feature.